dockerfile

```
@@ -0,0 +1,14 @@
+ARG PYTHON_VERSION=3.8
+FROM python:${PYTHON_VERSION}-slim AS base
+WORKDIR /app
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+
+FROM base AS final
+WORKDIR /app
+COPY . .
+RUN python manage.py migrate
+ENV PYTHONUNBUFFERED=1
```

```suggestion
ENV PYTHONUNBUFFERED=1
RUN python manage.py migrate
```

ENV should be added before execution of python command